everscale/caip2.md

```diff
@@ -0,0 +1,82 @@
+---
+namespace-identifier: everscale-caip2
+title: EVERSCALE namespace
+author: Lev Antropov(@levantropov), Vitaly Gritsay(@vvismaster)
+discussions-to: https://github.com/ChainAgnostic/namespaces/pull/52/
+status: Draft
+type: Standard
+created: 23.01.23
+requires (*optional):
+replaces (*optional):
+---
+
+# CAIP-2
+
+*For context, see the [CAIP-2](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-X.md) specification.*
+
+## Rationale
+`Mainnet` - main network of Everscale.
+
+`Devnet` - Evercloud Developer Network runs
+the latest stable version of Everscale Blockchain.
+The goal of Evercloud Devnet is to provide
+stable development environment for application developers.
+
+`Testnet(FLD)` - early network feature testing.
+
+However, the names of these networks
+are not used inside queries, they are only names.
+To work in a certain network, you only need
+to apply for the appropriate route(network endpoint).
+
+## Semantics and Syntax
+Based on the above, the syntax for constructing
+a namespace is a conditional arrangement,
+necessary rather for third-party projects
+(for example, WalletConnect, etc).
+```
+everscale:mainnet
+everscale:devnet
+```
+
+### Resolution Mechanics
+A transaction is the result of a contract execution.
+
+In general, a transaction is generated with one incoming message (external or internal) and can generate several outcoming messages (external or internal) as a result.
+
+The transaction can be successful or aborted.
+
+For example, a simple value transfer consists of 2 transactions
+— Sender's transaction which generated
+an internal message with a value transfer,
+and Recipient's transaction where it received
+the message with value and updated its balance.
```

Comment:
I'm sorry, I'm not sure this section is relevant to CAIP-2 (it might be to a future CAIP we're working on!).  This section is supposed to tell a non-everscale wallet how to confirm that they are communicated to the correct endpoints for "mainnet" or "devnet".  You mention above that CAIP-2 identifiers map to the URLs of specific endpoints, so I'm assuming from the test case below that "mainnet" maps to `https://dapp01.itgold.io/graphql`. -- but **where** is that map authoritatively hosted? Is there a machine-readable registry somewhere that can be queried when a dapp tells a wallet to connect to `everscale:unfamiliarsubnet`? Should they by default not trust any CAIP-2 not listed in this profile, making THIS the authoritative registry/mapping?

The ### Resolution Mechanics section should tell the developer how to query `https://dapp01.itgold.io/` for confirmation that it is, in fact, `mainnet`-- many other RPC specifications include an `info` or `help` command which returns this identifier, while many that don't encourage wallets to query for the hash of block 0 or block 1 and use that as confirmation.  Please update this section to include that kind of guidance.

Reply:
Thanks, we update this part.